release_notes/topics/17_1_0.adoc

```diff
@@ -0,0 +1,20 @@
+= Highlights
+
+== title
+
+lorem ipsum...
+
+== Updates
+
+A lot of effort improving the Quarkus distribution to make it as good as an experience as possible. We want to thank all of you for your feedback, and we are working to make the experience even better in the future.
+
+A few highlights include:
+
+* Upgraded Quarkus to 2.7.2.Final
+* Better UX when using database vendors that do not support the XA transaction mode (e.g. Galera, Azure managed SQL). Now you
+only set `db=<db>` and `transaction-xa-enabled=false` to use a non-XA JDBC driver with non-XA transaction handling.
+* Initial support for JSON log output using `log-json-enabled` configuration property
+* ...
+
+== Other improvements
+dolor sit amet
```

Comment:
Please see previous files for reference on how to format this file. It's included in release blog as well as docs, so need to be formatted accordingly.

There should be a highlights, with subsections, and if relevant an other improvements. There must not be placeholders though as the file should be "ready to be released" always ;)

Reply:
Just didn't think it's a highlight, so I didn't know what to write in the "highlights" section. Also not sure if we really want to promote it, as for me it is not clear galera/non-xa databases have first class support from our side. any idea? Should I just add the part around transactions in the highlights section instead?